hashes/Cargo.toml

```diff
@@ -15,16 +15,15 @@ exclude = ["tests", "contrib"]
 
 [features]
 default = ["std"]
-std = ["alloc", "bitcoin-io?/std", "hex/std"]
-alloc = ["bitcoin-io?/alloc", "hex/alloc"]
+std = ["alloc", "hex/std"]
+alloc = ["hex/alloc"]
```

Comment:
Should've been `hex?/`. This was broken already but maybe slipping the change in if you're going to rebase would be worth it.